resources_pages/install_ds_stack_windows.md

```diff
@@ -426,14 +426,21 @@ Click "OK" on all of the three windows we opened above and you're done! If you o
 both applications should return the same values, and the first one should be a path inside your user directory e.g.
 
 ```
-"C:/Users/joelo/R/win-library/4.0"   "C:/Program Files/R/R-4.0.2/library"
+"C:/Users/florencia/R/win-library/4.1" "C:/Program Files/R/R-4.1.0/library"
 ```
 
 If they don't return the same paths, please try to setting up your environmental variable again
 and making sure that it is pointing to the correct folder.
 
 **Do not continue unless both R from terminal and R from RStudio return the same paths here or later parts of the installation will fail.**
 
+Now we are going to change RStudio’s *Insert Pipe* shortcut so that it inserts the [new native pipe operator `|>`](https://blog.rstudio.com/2021/06/09/rstudio-v1-4-update-whats-new/). 
```

Comment:
Could you make changes like this in all three instructions (mac, win, linux?), so that they don't go out of sync with eachother and we forget one of them.